Documentation/doc/scripts/html_output_post_processing.py

```diff
@@ -57,7 +57,7 @@ def write_out_html(d, fn):
     f.write('<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">\n')
     f.write('<html xmlns=\"http://www.w3.org/1999/xhtml\">')
     if d.html() is not None:
-      f.write(d.html(method='html'))
+      f.write(d.html().replace("&#13;",""))
```

Comment:
we can't remove the html method otherwise you get duplicated output (I don't remember the reason) but I can reproduce the duplicated output without it.

Reply:
@sloriot 
Where do you see the duplicated output when using the adjusted version?

Reply:
In `Manual/how_to_cite_cgal.html`, I got all entries being duplicated in another bloc at the end of the file.

Reply:
I think I can propose at alternative fix, if the issue is only in this file.

Reply:
I see here a nice table with 120 items (see https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.1-Ic-134/output2/Manual/how_to_cite_cgal.html and https://cgal.geometryfactory.com/CGAL/Manual_doxygen_test/CGAL-6.1-Ic-134/master/Manual/how_to_cite_cgal.html) but I don't see duplicated entries.

Reply:
Regarding:
> I think I can propose an alternative fix, if the issue is only in this file.

what are you thinking about?